### ISSUE TITLE
LSTCam-001 file has bad headers, breaking ctapipe tests

### ISSUE THREAD

**Issue by kosack** [cta-observatory/ctapipe-extra#13]
in ctapipe-extra 0.2.15 and above, it seems some bad headers in `LSTCam-001.camgeom.fits.gz` break some tests.  

the bad headers are:

```
CAM_ID = 'LST1Cam'  # should be 'LSTCam-001'
PIX_ROT = 0.0       # should be some value that makes the hexagons appear in the correct alignment
```

**Comment by kosack** [comment]
see https://github.com/cta-observatory/ctapipe/issues/813

**Comment by FrancaCassol** [comment]
I understand, I will produce a correct LSTCam-001.camgeom.fits.gz, and pull it after test.

**Comment by maxnoe** [comment]
solved in v0.2.16
